contracts/src/tokens/memecoin.cairo

```diff
@@ -375,6 +361,11 @@ mod UnruggableMemecoin {
             if !self.is_launched() {
                 self.enforce_prelaunch_holders_limit(sender, recipient, amount);
             } else {
+                if (get_caller_address() == self.pair_address.read()
+                    || recipient == self.pair_address.read()) {
+                    return;
+                }
+
```

Comment:
Is it the pair that we'd like to whitelist or the router ?

Reply:
the pair, it's the pair that calls transferFrom(swapper, self, amount)